src/features/island/landscaping/LandscapingGrid.tsx

```
@@ -0,0 +1,88 @@
+import React, { useContext, useEffect, useState } from "react";
+import classNames from "classnames";
+import { useSelector } from "@xstate/react";
+
+import { Context } from "features/game/GameProvider";
+import { GRID_WIDTH_PX } from "features/game/lib/constants";
+import type { MachineState as GameMachineState } from "features/game/lib/gameMachine";
+import type { MachineInterpreter } from "features/game/expansion/placeable/landscapingMachine";
+
+const _landscaping = (state: GameMachineState) =>
+  state.matches("landscaping");
```

ESLint

[error] Delete `⏎·`

(prettier/prettier)